src/prelude.rs

```diff
@@ -12,14 +12,13 @@
 
 pub use crate::err::{PyErr, PyResult};
 pub use crate::gil::GILGuard;
-pub use crate::instance::{AsPyRef, Py, PyRef, PyRefMut};
+pub use crate::instance::{AsPyRef, Py};
 pub use crate::object::PyObject;
 pub use crate::objectprotocol::ObjectProtocol;
 pub use crate::python::Python;
 pub use crate::{FromPy, FromPyObject, IntoPy, IntoPyPointer, PyTryFrom, PyTryInto, ToPyObject};
 // This is only part of the prelude because we need it for the pymodule function
+pub use crate::pyclass_init::PyClassInitializer;
 pub use crate::types::PyModule;
```

Comment:
This new `pub use crate::pyclass_init::PyClassInitializer;` has split up a comment from the line beneath it.

Reply:
:+1: